metricflow/cli/main.py

```diff
@@ -695,6 +697,19 @@ def validate_configs(cfg: CLIContext, dw_timeout: Optional[int] = None, skip_dw:
     """Perform validations against the defined model configurations."""
     cfg.verbose = True
 
+    lint_spinner = Halo(text="Checking for yaml lint struture issues", spinner="dots")
```

Comment:
Nit: Maybe `Checking for YAML format issues`, `Checking for YAML lint issues`, etc.